tests/transforms/test_suite2p_wrapper.py

```diff
@@ -121,8 +122,8 @@ def test_suite2p_wrapper(
     s = suite2p_wrapper.Suite2PWrapper(input_data=args, args=[])
     s.run()
 
-    if bin_size is not None:
-        assert s.args['nbinned'] == int(nframes / bin_size)
+    if movie_frame_rate is not None:
+        assert s.args['nbinned'] == int(nframes / (3.7 * movie_frame_rate))
```

Comment:
3.7 here could be replaced by `s.args['bin_duration']` pulling the default value from the args.

Reply:
Good idea. Done!